trunkfingerprint--1.2.0.sql

```diff
@@ -90,6 +89,13 @@ begin
                                                                 then '(select oprname from pg_operator where oid = foo.' || attname || ')'
                                  when attname like '%opc'       then '(select opcname from pg_opclass where oid = foo.' || attname || ')'
                                  when attname like '%tablespace'then '(select spcname from pg_tablespace where oid = foo.' || attname || ')'
+                                 when attname like '%partdefid' then '(select relname from pg_class where oid = foo.' || attname || ')'
```

Comment:
We should rather add it to those below at line 93 (original numeration). Rationale:
1) output schema-qualified table name, not just the table name itself
2) reduce code bloating